mod/01_rating/set/abstract/all_values.rb

```diff
@@ -0,0 +1,182 @@
+include_set Type::SearchType
```

Comment:
nevermind.  all_values is fine.  I thought there was a distinction being made between "all values" and "all metric values", but if there is, I'm not understanding it yet.